### PR TITLE
Allow org unit ingestion

### PR DESCRIPTION
**Pull request by mreaugusto**

### CODE REVIEW

File: python/validator/schema.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
       "event_version": {
         "type": "string",
         "required": true,
-        "allowed": ["1.4","1.5"]
+        "allowed": ["1.4","1.5", "1.6"]
```
Comment (amlanc1) [inline python/validator/schema.json:14]: This change is not needed. This file is specific to DF application. Not related to BI adapter

File: python/validator/schema.json [hunk 2]
```diff
@@ -52,6 +52,14 @@
...
             "required": true
           },
+          "enforcement_org_unit_cd": {
+            "type": "string",
+            "required": true
+          },
+          "enforcement_org_unit_cd_txt": {
+            "type": "string",
+            "required": true
+          },
```
Comment (amlanc1) [inline python/validator/schema.json:62]: Same for this. Not needed to change this file

**Review by amlanc1 — CHANGES_REQUESTED** [review]
Check the comments

File: python/validator/schema.json (hunk 1 above)
Comment (mreaugusto) [reply python/validator/schema.json]: Amlan, this is also used by the validator to validate the message pushed to the queue

File: python/validator/schema.json (hunk 2 above)
Comment (mreaugusto) [reply python/validator/schema.json]: Same as above

**Pull request closed without merging**